client/src/components/Game/FabricCanvas.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect } from 'react';
+import { FabricContext } from './FabricContextProvider';
+import { fabric } from 'fabric';
+import { saveAs } from 'file-saver';
+
+const FabricCanvas = props => {
+    const [canvas, initCanvas] = useContext(FabricContext);
```

Comment:
Nit: Spacing inconsistent?
```suggestion
const FabricCanvas = props => {
  const [canvas, initCanvas] = useContext(FabricContext);
```

Reply:
OMG U ACTUALLY COMMENTED LOL
i didnt know u could do suggestions in comments thank u